icu4c/source/i18n/unicode/unum.h

```diff
@@ -1228,10 +1249,12 @@ unum_getDoubleAttribute(const UNumberFormat*          fmt,
 /**
 * Set a numeric attribute associated with a UNumberFormat.
 * An example of a numeric attribute is the number of integer digits a formatter will produce.
-* If the formatter does not understand the attribute, this call is ignored.
+* If the formatter does not understand the attribute, this call is ignored.  (The caller can use
+( unum_hasAttribute() to tell in advance whether the formatter understands the attribute.)
```

Comment:
Nit, here and elsewhere: I don't think you should or need to put the docs in parentheses, because (1) this is an important part of the documentation, and (2) the parentheses may render oddly.